packages/webawesome/docs/_transformers/anchor-headings.js

```diff
@@ -79,7 +79,7 @@ export function anchorHeadingsTransformer(options = {}) {
       const anchor = parse(`
         <a href="#${encodeURIComponent(id)}">
           <span class="wa-visually-hidden"></span>
-          <span aria-hidden="true">#</span>
+          <wa-icon variant="regular" name="hashtag" aria-hidden="true" class="icon-shrink"></wa-icon>
```

Comment:
```suggestion
          <wa-icon variant="regular" name="hashtag" class="icon-shrink"></wa-icon>
```

Because we're omitting the `label=""` attribute, `aria-hidden="true"` gets added by the component automatically.